exampleSite/config.yaml

```diff
@@ -16,11 +16,6 @@ params:
   # Enable float footnotes.
   # Default to true
   enableFootnotes: true
-
-  # Enable line numbers, it need to set the same value with
-  # markup.hightlight.lineNos
-  # Default to false
-  enableLineNos: true
```

Comment:
I think the `exampleSite/config.example.toml` still has a `enableLineNos` field

Reply:
Oh, thanks for the reminder.

Reply:
Removed in #1ec0b26